libs/initScripts/local.js

```diff
@@ -192,3 +230,36 @@ const _initAllVersions = async (audiusLibs) => {
     await setServiceVersion(audiusLibs, serviceType, serviceVersions[serviceType])
   }
 }
+
+const _updateDelegateOwnerWalletInDockerEnv = async (readPath, writePath, delegateOwnerWallet, delegateWalletPkey) => {
```

Comment:
looks like regardless of whether `delegateOwnerWallet` and `delegatePrivateKey` are found, you still add the respective values to the file? do we need the for loop check?

Reply:
discussed offline - need to update current line with `delegateOwnerWallet` and `delegatePrivateKey`with new value, and if dne, append to end of file.